daisy/step_wait_for_instances_signal.go

```diff
@@ -150,7 +151,20 @@ func waitForSerialOutput(s *Step, project, zone, name string, so *SerialOutput,
 				return Errf("WaitForInstancesSignal: instance %q: error getting serial port: %v", name, err)
 			}
 			start = resp.Next
-			for _, ln := range strings.Split(resp.Contents, "\n") {
+			lines := strings.Split(resp.Contents, "\n")
+			for i, ln := range lines {
+				// If there is a unconsumed tail string from the previous block of content, concat it with the 1st line of the new block of content.
+				if i == 0 && tailString != "" {
+					ln = tailString + ln
+					tailString = ""
+				}
```

Comment:
We're doing this concatenation (and serial port reading) in the CreateInstance step. Thoughts on consolidating? 

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/4e113aa505000dbc40935f5f3b93f47241fe565f/daisy/step_create_instances.go#L98

Reply:
as discussed offline, it's unnecessary since serial log itself doesn't have any pattern match; also it's non-trivial to make a common loop function here since the actions inside the loop are quite different. Leave it as-is for now.